tests/unit/util/test_logging.py

```diff
@@ -8,12 +12,42 @@
 from logprep.util.logging import LogprepFormatter
 
 
-class TestLogprepFormatter:
+def setup_module():
+    logging.config.dictConfig(DEFAULT_LOG_CONFIG)
+
+
+class TestLogDictConfig:
+    """this tests the logprep.util.defaults.DEFAULT_LOG_CONFIG dict"""
+
+    def test_console_logger_uses_logprep_formatter(self):
+        logger = logging.getLogger("console")
+        assert isinstance(logger.handlers[0].formatter, LogprepFormatter)
 
-    def test_default_log_config(self):
-        logging.config.dictConfig(DEFAULT_LOG_CONFIG)
-        logger = logging.getLogger("test")
-        assert isinstance(logger.root.handlers[0].formatter, LogprepFormatter)
+    def test_root_logger_has_only_quehandler(self):
+        logger = logging.getLogger("root")
+        assert isinstance(logger.handlers[0], logging.handlers.QueueHandler)
```

Comment:
according to the test title you also need one more assertions
```suggestion
        assert len(logger.handlers) == 1, "only one handler should be set"
        assert isinstance(logger.handlers[0], logging.handlers.QueueHandler)
```